modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -218,6 +218,41 @@ enum ImwriteHDRCompressionFlags {
 
 //! @} imgcodecs_flags
 
+/** @brief Represents an animation with multiple frames.
+The `Animation` struct is designed to store and manage data for animated sequences such as those from animated formats (e.g., GIF, APNG, WebP).
+It provides support for looping, background color settings, frame timing, and frame storage.
+*/
+struct CV_EXPORTS_W_SIMPLE Animation
+{
+    //! Number of times the animation should loop. 0 means infinite looping.
+    CV_PROP_RW int loop_count;
+    //! Background color of the animation in RGBA format.
```

Comment:
Background color of the animation in RGBA format.
@asmorkalov @vrabaud should it be BGRA ? btw WebP has ARGB

Reply:
Let it be BGRA as everywhere in OpenCV.

Reply:
Also you can use cv::Scalar.